tach.toml

```diff
@@ -0,0 +1,83 @@
+exclude = []
+source_roots = [
+    "model/atmosphere/advection/src",
+    "model/atmosphere/diffusion/src",
+    "model/atmosphere/dycore/src",
+    "model/atmosphere/subgrid_scale_physics/microphysics/src",
+    "model/common/src",
+    "model/driver/src",
+    "tools/src",
+]
+
+[[modules]]
+path = "icon4py.model.atmosphere.advection"
+depends_on = [
+    { path = "icon4py.model.common" },
+]
+
+[[modules]]
+path = "icon4py.model.atmosphere.diffusion"
+depends_on = [
+    { path = "icon4py.model.common" },
+]
+
+[[modules]]
+path = "icon4py.model.atmosphere.dycore"
+depends_on = [
+    { path = "icon4py.model.common" },
+]
+
+[[modules]]
+path = "icon4py.model.atmosphere.subgrid_scale_physics.microphysics"
+depends_on = [
+    { path = "icon4py.model.common" },
+]
+
+[[modules]]
+path = "icon4py.model.common"
+depends_on = []
+
+[[modules]]
+path = "icon4py.model.driver"
+depends_on = [
+    { path = "icon4py.model.atmosphere.diffusion" },
+    { path = "icon4py.model.atmosphere.dycore" },
+    { path = "icon4py.model.common" },
+]
+
+[[modules]]
+path = "icon4pytools"
```

Comment:
For the tools, I don't know what kind of restriction we should have. Most of them look reasonable to me. In the end, they are all in the same package so at least you never end up installing something that cannot be built because it imports a missing dependency. This would currently be the case if we imported something from `model.atmosphere.diffusion` in `model.atmosophere.dycore`

Reply:
Just FYI, the current dependencies are the ones that the tool automatically detects in the codebase.